pkg/cloud/cloud.go

```diff
@@ -1044,3 +1052,14 @@ func volumeModificationDone(state string) bool {
 	}
 	return false
 }
+
+func getVolumeAttachmentsList(volume *ec2.Volume) []string {
+	var volumeAttachmentList []string
+	if len(volume.Attachments) > 0 {
```

Comment:
Do you need this check?

Reply:
You are right, we actually not need this.